src/walletSdk/horizon/Stellar.ts

```diff
@@ -1,6 +1,13 @@
-// TODO - https://stellarorg.atlassian.net/browse/WAL-814?atlOrigin=eyJpIjoiMWY5MjBkZTE4MTg3NDA3N2E0MjYwMmQ2ZmRhOGFiODUiLCJwIjoiaiJ9
+import { AccountService } from "./AccountService";
 
 // Do not create this object directly, use the Wallet class.
 export class Stellar {
-  constructor(cfg) {}
+  private cfg;
```

Comment:
could we add type for `cfg` ?

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/21/commits/44202897d86f2a6b9f27384178bfdae68b9d4646